### ISSUE TITLE
About the alpha value

### ISSUE THREAD

**Issue by hbin0701** [sarthak268/c3vqg-official#6]
Hello :)
Thank you for sharing this amazing code!

As I'm running this code in colab (python3), and  I'm changing several things to accomodate to my settings.
Everything works fine until now, except for when running script for training:

in /models/iq.py ,
self.alpha = nn.Parameter(torch.randn(z_size)).  # line 82
and line 119 error occurs,
   eps = Variable((Normal(torch.zeros_like(mu).cuda(), self.alpha.data.pow(-1))).sample())

I realized STD in Normal distribution should be positive, and along with this, gradient suddenly explodes, and loss becomes nan.

Thus, I introduced following lines of codes.

# Replace line 119 with:
d = self.alpha.data.pow(-1)
d = torch.nan_to_num(d.clamp(min=1e-4, max=2), 1e-4) # using 2 instead of 1e-4 causes gradient explosion.
eps = Variable((Normal(torch.zeros_like(mu).cuda(), d)).sample())

Do you think that this implementation would not affect your intentions introduced in the paper?
Thank you in advance!

**Comment by mahmudhasankhan** [comment]
@hbin0701 hello! Were you able to fix the gradient explosion part?
